src/extension/csw/web-csw/src/main/java/org/geoserver/csw/web/CSWServiceDescriptionProvider.java

```diff
@@ -36,6 +37,11 @@ public CSWServiceDescriptionProvider(GeoServer gs) {
         catalog = gs.getCatalog();
     }
 
+    @Override
+    public List<String> getServiceTypes() {
+        return Collections.singletonList(SERVICE_TYPE);
```

Comment:
Two bits that I don't consider in actual need of fixing, but give me pause so I'll share a comment anyways:

* I see this code repeated over and over... feels like this should be a field of the class, provided via constructor, and then have a getter in the base class
* Using ``List.of(SERVICE_TYPE)`` would be the Java11+ way to achieve the same goad (all types of immutable lists have basically been replaced by ``<MyCollectionType>.of(xyz)``

Reply:
thanks for feedback, resolved.